docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,216 @@ title: Architecture
+              {
+                "CAFAuditInteger": {
+                  "match": "*_CAInt",
+                  "mapping": {
+                    "type": "integer"
+                  }
+                }
+              },
+              {
+                "CAFAuditShort": {
+                  "match": "*_CAShort",
+                  "mapping": {
+                    "type": "short"
+                  }
+                }
+              },
+              {
+                "CAFAuditDouble": {
+                  "match": "*_CADbl",
+                  "mapping": {
+                    "type": "double"
+                  }
+                }
+              },
+              {
+                "CAFAuditFloat": {
+                  "match": "*_CAFlt",
+                  "mapping": {
+                    "type": "float"
+                  }
+                }
+              },
+              {
+                "CAFAuditDate": {
+                  "match": "*_CADte",
+                  "mapping": {
+                    "type": "date"
+                  }
+                }
+              },
+              {
+                "CAFAuditBoolean": {
+                  "match": "*_CABln",
+                  "mapping": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            ],
+            "properties": {
+              "applicationId": {
+                "type": "keyword"
+              },
+              "correlationId": {
+                "type": "keyword"
+              },
+              "eventCategoryId": {
+                "type": "keyword"
+              },
+              "eventOrder": {
+                "type": "long"
+              },
+              "eventTime": {
+                "type": "date"
+              },
+              "eventTimeSource": {
+                "type": "keyword"
+              },
+              "eventTypeId": {
+                "type": "keyword"
+              },
+              "processId": {
+                "type": "keyword"
+              },
+              "threadId": {
+                "type": "long"
+              },
+              "userId": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      }
+    }
+
+The above JSON, returned from Elasticsearch, illustrates the field type mappings for a tenant's index. Audit event information is stored in fixed fields and the event parameters are mapped to appropriate types based on field name suffixes added by the Audit library.
+
+A tenant application's audit events are sent from the client-side library to Elasticsearch and added to the index created for the tenant.
+
+    GET /1_audit/cafAuditEvent/_search
+    {
+      "took": 3,
+      "timed_out": false,
+      "_shards": {
+        "total": 5,
+        "successful": 5,
+        "failed": 0
+      },
+      "hits": {
+        "total": 2,
+        "max_score": 1,
+        "hits": [
+          {
+            "_index": "1_audit",
```

Comment:
"On the audited application's first call to the Audit library, an index is created for the tenant. A tenant's Elasticsearch index, type and document meta-field identifiers (`index/type/doc`) are seen as `<tenantId>_audit/cafAuditEvent/applicationAuditEvent`." this to me describes the naming of indices. Do you think it would be better if I just replaced all instances of "1_audit" with "<tenantId>_audit" ?